### ISSUE TITLE
Tool to find yarn.lock files on GitHub

### ISSUE THREAD

**Issue by moretea** [moretea/yarn2nix#18]
We want to see if we can find packages that break with `yarn2nix`. This tool should do something similar to crater from rust-lang.

**Comment by manveru** [comment]
https://github.com/search?utf8=%E2%9C%93&q=filename%3Ayarn.lock&type=Code&ref=searchresults

**Comment by moretea** [comment]
Yes indeed. Last night,  I created a small program that does that via the API, downloads the first 1000 hits and downloads the `yarn.lock` and `package.json` files in those repositories.

**Comment by jluttine** [comment]
Here's one non-working example: https://github.com/input-output-hk/daedalus

It breaks because some dependencies in yarn.lock are git+https. Any workaround?

**Comment by srghma** [comment]
@jluttine this pr https://github.com/moretea/yarn2nix/pull/92 adds support for `git-https`
